src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import './variable';
+
+.page {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    height: $minutes-height;
+    width: $minutes-width;
+    background-color: $minutes-color;
+    bottom: 50%;
+    left: calc(50% - (#{$minutes-width} / 2));
+    transform-origin: bottom center;
+    animation: rotate 3600s steps(60) infinite;
+    animation-duration: 600s;
+  }
+
+  &__seconds {
+    position: absolute;
+    height: $seconds-height;
+    width: $seconds-width;
+    background-color: $seconds-color;
+    bottom: 50%;
+    left: calc(50% - (#{$seconds-width} / 2));
+    transform-origin: bottom center;
+    animation: rotate 60s linear infinite;
+    animation-duration: 10s;
```

Comment:
There is a discrepancy in the animation duration for the '.stopwatch__seconds' class. The 'animation' shorthand property specifies '60s', but the 'animation-duration' property specifies '10s'. Ensure these values are consistent to avoid unexpected behavior.